.github/workflows/test.yml

```
@@ -61,27 +53,19 @@ jobs:
-        uses: erlef/setup-beam@v1
+      - name: Common Elixir Setup for Lint
+        uses: ./.github/actions/common-elixir-setup
         with:
           elixir-version: "1.18"
           otp-version: "27"
-
-      - name: Restore dependencies cache
-        uses: actions/cache@v3
-        with:
-          path: |
+          cache-key-prefix: mix-lint
```

actionlint (1.7.4)

input "cache-key-prefix" is not defined in action "Common Elixir Setup" defined at "./.github/actions/common-elixir-setup". available inputs are "elixir-version", "otp-version"

(action)